docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -39,12 +39,7 @@ killed for this purpose.
 
 ## Config
 
-Rescheduler should be [enabled by default as a static pod](https://git.k8s.io/kubernetes/cluster/saltbase/salt/rescheduler/rescheduler.manifest).
-It doesn't have any user facing configuration (component config) or API and can be disabled:
-
-* during cluster setup by setting `ENABLE_RESCHEDULER` flag to `false`
-* on running cluster by deleting its manifest from master node
-(default path `/etc/kubernetes/manifests/rescheduler.manifest`)
+Rescheduler should be enabled. If doesn't have any user facing configuration (component config) or API and can be disabled too.
```

Comment:
"If doesn't have..." sounds like it's missing a word in there, perhaps "If the rescheduler doesn't have"? I'm not entirely sure what that sentence is trying to convey.

Is the information you deleted still relevant here? It seems extraneous from the saltbase removal, and looks like it might have been an accidental deletion.

Reply:
Because `/cluster/saltbase` directory removed, some link link to this directory should update too.  Besides,  `saltbase`s function  should be replaced by `kubernete-anywhere`, so export `ENABLE_RESCHEDULER` should removed too.